packit_service/worker/handlers/copr.py

```diff
@@ -564,10 +565,13 @@ def handle_scan(self):
                 logger.debug("Something went wrong, skipping the reporting.")
                 return
 
-            logger.info(f"Scan submitted successfully, output: {output}")
+            logger.info("Scan submitted successfully.")
 
-            response = json.loads(output)
-            if not (url := response.get("url")):
+            response_dict = self.parse_dict_from_output(output)
+
+            logger.debug(f"Parsed dict from output: {response_dict} ")
+
+            if not (url := response_dict.get("url")):
                 logger.debug("It was not possible to get the URL from the response.")
```

Comment:
It should probably be an `error`.

Reply:
we are sending those to Sentry, so it could cause a lot of noise, therefore I prefer keeping it as is